server/src/service.rs

```diff
@@ -35,8 +36,15 @@ impl Service {
         let indexify_state = IndexifyState::new(self.config.state_store_path.parse()?).await?;
         let blob_storage = Arc::new(BlobStorage::new(self.config.blob_storage.clone())?);
         let executor_manager = Arc::new(ExecutorManager::new(indexify_state.clone()).await);
+        let object_store = self.config.blob_storage.object_store()?;
+        let kvs_manifest_path = Path::new(&self.config.state_store_path).join("graph_ctx_state");
+        let kvs_manifest_path = kvs_manifest_path
+            .to_str()
+            .ok_or(anyhow!(format!("unable to create kv store pathl")))?;
+        let kvs = KVS::new(object_store.clone(), kvs_manifest_path).await?;
```

Comment:
Instead of getting the object_store from blob_storage config, can we make KVS support getting a BlobStorage instance and from it, get the object_store?

```suggestion
        let kvs_manifest_path = kvs_manifest_path
            .to_str()
            .ok_or(anyhow!(format!("unable to create kv store pathl")))?;
        let kvs = KVS::new(blob_storage.clone(), kvs_manifest_path).await?;
```

Reply:
@seriousben Updated the logic here, lmk what you think. Simplified the config to have only the path, and from there create an object store object.